### ISSUE TITLE
Accessibility - Keyboard user is unable to get the tooltips which are shown using mouse hover on Grape editor but not appearing using tab navigation

### ISSUE THREAD

**Issue by sagaranandani17** [artf/grapesjs#2390]
**OS version:** 1903 (OS Build: 18326.295)
**Browser:** Edge(Microsoft Edge 44.18362.267.0)

**User Experience:** 
If tooltips are not accessible with keyboard inside editor, than keyboard user will not be able to understand the control purpose and will face difficulties while interacting with it.

**Steps to reproduce**
When user moves to different components using mouse hover, the name of component is shown as tooltip to user (For image component image is shown, for Text component text is shown), but when user navigate to different components using keyboard then tooltip is not shown. 

**Actual Result:**
Keyboard user is unable to get the tooltips which are shown using mouse hover but not appearing using tab navigation.

**Expected Result:**
Keyboard user should be able to see the corresponding tooltip information when he/she navigates on to different components in editor.

**Please find the attached image**
Tooltip shown on mouse hover
![image](https://user-images.githubusercontent.com/57341535/68465406-6f7b5800-0238-11ea-9790-dd3c678a3269.png)

**Comment by artf** [comment]
I'd point all accessibility issues here #654
